backend/docker-services/keycloak/docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: '3.9'
+
+services:
+  keycloak-db:
+    image: postgres:14
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: keycloak
+      POSTGRES_USER: keycloak
+      POSTGRES_PASSWORD: keycloak
+    networks:
+      - keycloak-dev
+
+  keycloak:
+    depends_on:
+      - keycloak-db
+    container_name: keycloak_dev
+    command: start-dev --import-realm
+    environment:
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
```

Comment:
Is there any risk of this being accidentally deployed like this? Maybe we should require the env to be set to a more secure value?

Reply:
No there is no risk since we don't use docker compose for deployment. We have a single keycloak for all apps at Maykin, and the municipalities would have their own. This is really just to make sure that the flow works.

Reply:
Discussed: 
- Add comment that docker compose is only for dev
- Remove Keycloak admin user
- Set in the docs that you can add an admin user